package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@outreach/extensibility-sdk",
+  "name": "@getoutreach/extensibility-sdk",
   "license": "MIT",
   "version": "0.9.12",
```

Comment:
Let's start the new package with a proper versioning: `1.0.0`

Reply:
changes